config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   root 'articles#index'
 
+  get '/articles', to: 'articles#index'
```

Comment:
This doesn't belong here. It's part of https://github.com/lprogg/Blog/pull/1.

Reply:
I added /articles route later during development, when I also implemented the route parameter :id.